contrib/systemd/etcd2-backup/README.md

```diff
@@ -0,0 +1,65 @@
+# etcd2-backup
+
+Remote backup and multi-node restore services for etcd2 clusters.
+
+## Terminology
+
+**founding member** : The node which is the first member of the new recovered cluster. It is this node's rclone backup data (only) that will be used to restore the cluster. The rest of the nodes will join the cluster with no data, and simply catch up with the **founding member**.
+
+## Configuration
+Before you get started, you need to configure `30-etcd2-backup-restore.conf`
+
+```
+[Service]
+Environment="ETCD_RESTORE_MASTER_ADV_PEER_URLS=<http://host:port>"
+Environment="RCLONE_ENDPOINT=endpoint-name:path/to/backups"
+```
+
+Assuming you're deploy to CoreOS with etcd2, you should only need to change
+
+- `ETCD_RESTORE_MASTER_ADV_PEER_URLS`
+   This is the advertised peer url of the etcd2 node that will be the founding member of the new restored cluster. We will call this node the **founding member**.
+
+-  `RCLONE_ENDPOINT`
+    The rclone endpoint to which backups will be stored. See rclone docs for more information
+
+    Feel free to point any number of machines as the same RCLONE_ENDPOINT, path and all. Backups for each machine are stored in a sub-folder named with the machine ID (%m in systemd parlance)
+
+-  `./rclone.conf`
+    The rclone configuration file which will be installed. Must list a [section] which matches `RCLONE_ENDPOINT`'s endpoint-name component.
+
+If you want to adjust backup frequency, edit `./etcd2-backup.timer`
+
+## Installation
+Once you've got those things configured, you can run `./build`.
+
+That script will generate a tarball  can be copied to CoreOS instances and contains a script `etcd2-backup-install`
+
+After the tarball is inflated and the install script exected, you should see three new systemd services
+
+- `etcd2-backup.service`
+   A oneshot service which calls `etcdctl backup` and syncs the backups to the rclone endpoint (using an rclone container, of course). `etcd2-backup.timer` is responsible for periodically running this service.
```

Comment:
Might be good to include the systemctl commands for running/enabling the backup services (and timer)